custom_components/smart_maic/const.py

```diff
@@ -5,6 +5,7 @@
 DOMAIN = "smart_maic"
 PREFIX = "smart-maic"
 HTTP_TIMEOUT = 5
+EXPIRATION_INTERVAL = 120
 
 IP_ADDRESS = CONF_IP_ADDRESS
```

Comment:
it would be great to have the ability to configure EXPIRATION_INTERVAL via UI... :)

Reply:
Why not, added, please test

Reply:
awesome! 🤗
![Screenshot_2024-12-05-17-57-09-222_io.homeassistant.companion.android-edit.jpg](https://github.com/user-attachments/assets/304ae5e1-4b1d-465f-83d7-1d8a359243bf)